executioncontext/projectview/src/main/kotlin/org/jetbrains/bsp/bazel/projectview/model/ProjectView.kt

```diff
@@ -32,6 +33,8 @@ data class ProjectView(
     val deriveTargetsFromDirectories: ProjectViewDeriveTargetsFromDirectoriesSection?,
     /** level of depth for importing inherited targets */
     val importDepth: ProjectViewImportDepthSection?,
+    /** manually enabled rules to overcome false negatives*/
```

Comment:
I would change the description to `manually enabled rules to override the automatic rules detection mechanism`.